test/serving-e2e/testdata/serving-pipeline-cat.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: serving-source
+spec:
+  vertices:
+    - name: serving-in
+      servingStoreName: default
+      scale:
+        min: 1
+        max: 1
+      source:
+        serving:
+          service: true
+          msgIDHeaderKey: "X-Numaflow-Id"
+
+    - name: cat
+      scale:
+        min: 1
+        max: 1
+      udf:
+        container:
+          image: quay.io/numaio/numaflow-go/map-forward-message:stable
+
+    - name: serve-sink
+      servingStoreName: default
+      scale:
+        min: 1
+        max: 1
+      sink:
+        udsink:
+          container:
+            image: quay.io/numaio/numaflow-go/sink-serve:stable
```

Comment:
Now that sink-serve is added to e2e, we want to make sure numaflow CI always uses the latest `stable` image. Numaflow-go supports auto-updating the stable image whenever there is a merge from numaflow-go. To enable it, please add the serve sink [here](https://github.com/numaproj/numaflow-go/blob/main/.github/workflows/build-push.yaml#L18) . Thanks!

Reply:
nvm, just saw it in this pr. https://github.com/numaproj/numaflow-go/pull/182/files